service/service.go

```diff
@@ -62,7 +69,9 @@ func (s *Service) StartSubscriptions(events ...event.Event) error {
 }
 
 func (s *Service) Close() error {
-	// TODO close query client
+	if err := s.queryClient.Close(); err != nil {
```

Comment:
Oh, it's already outputting a warning log. If so, it looks like you don't need a return value.